crypto/src/hash.rs

```diff
@@ -94,65 +113,67 @@ impl<T> fmt::Debug for HashOf<T> {
 }
 
 impl<T> Clone for HashOf<T> {
+    #[inline]
```

Comment:
This doesn't add much; rust generics are templated so the function is implicitly inline.

Reply:
> Besides #[inline], there are two more exceptions to this. Generic functions are implicitly inlinable. Indeed, the compiler can only compile a generic function when it knows the specific type arguments it is instantiated with. As that is known only in the calling crate, bodies of generic functions have to be always available.

Credit https://matklad.github.io/2021/07/09/inline-in-rust.html

Reply:
I get you, won't do that anymore